plugins/request-uri-manipulations.rst

```diff
@@ -0,0 +1,25 @@
+Request URI Manipulations
+=========================
+
+Request URI manipulations can be done thanks to several plugins:
+
+* ``AddHostPlugin``: Sets default host, scheme and port of each request.
+* ``AddPathPlugin``: Sets default base path of each request.
+* ``BaseUriPlugin``: Is a combination of ``AddHostPlugin`` and ``AddPathPlugin`` using the ``UriInterface`` class.
```

Comment:
lets leave away the mention of UriInterface, that is just an implementation detail